generator/internal/descriptor_utils.cc

```diff
@@ -179,11 +179,11 @@ void SetMethodSignatureMethodVars(
   for (int i = 0; i < method_signature_extension.size(); ++i) {
     google::protobuf::Descriptor const* input_type = method.input_type();
     std::vector<std::string> parameters =
-        absl::StrSplit(method_signature_extension[i], ',');
+        absl::StrSplit(method_signature_extension[i], ',', absl::SkipEmpty());
```

Comment:
`SkipEmpty` seems like something you wouldn't want to do in a comma-separated list of names.  Is that really OK?